src/dlob.rs

```diff
@@ -43,14 +44,23 @@ impl DLOBClient {
     }
     /// Query L2 Orderbook for given `market`
     pub async fn get_l2(&self, market: MarketId) -> Result<L2Orderbook, SdkError> {
+        self.get_l2_with_depth(market, DEFAULT_ORDERBOOK_DEPTH)
```

Comment:
to avoid confusion it should return full depth by default as its unclear what the right value for depth should be otherwise.
suggest change the api like this:
```suggestion
    pub async fn get_l2(&self, market: MarketId, depth: Option<u8>) -> Result<L2Orderbook, SdkError> {
```

Reply:
Well, I initially thought of this approach, but then decided against it, as it will be a breaking change in API. But if that's acceptable, I can rewrite it to use optional depth, without extra method. @jordy25519

Reply:
I think here the breaking change is acceptable, we're still very much incubating this crate

Reply:
Ok, I've rewritten the original method to accept extra optional depth parameter.